cmd/cloud/group.go

```diff
@@ -93,12 +93,7 @@ var groupCreateCmd = &cobra.Command{
 			return errors.Wrap(err, "failed to create group")
 		}
 
-		err = printJSON(group)
-		if err != nil {
-			return err
-		}
-
-		return nil
+		return printJSON(group)
```

Comment:
I thought this (this whole hunk) was against our style guide for some reason..

Reply:
I think I vaguely remember talking about this before and hope I wasn't confusing with my thoughts at the time. Basically, we don't do this normally because we try to wrap errors almost always. Because there was no wrapping I just shortened it here.

That said, I probably shouldn't have done it and instead wrapped the error.

Reply:
Yeah I definitely got confused before; thanks for the clarification of the style rule!

Reply:
No worries! Definitely call me out if I am saying one thing and doing another.